notebooks/notebooks/iclr2022/estimate-gpu-days-adv.ipynb

```diff
@@ -0,0 +1,321 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.    gpu_type_to_game_throughputs = {}
</pre><p>Could simplify the code below by using a defaultdict(list).</p><p>See https://docs.python.org/3/library/collections.html#defaultdict-examples</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlignmentResearch/KataGoVisualizer/pull/62/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
thanks, I didn't know about this class

Reply:
Learned about it way back when I interned at Dropbox and a senior person told me about it during code review haha